### ISSUE TITLE
Group work: Planning Database

### ISSUE THREAD

**Issue by tguadalupe** [tguadalupe/UVI-Dining#5]
So far it seems we have 3 tables. However as Kentopp mention in class about one to many etc. I believe we might need more tables and to take out Breakfast l,unch and dinner and make a next table. 

**From excel sheet:**

| Admin  |
| ------------- | 
| **PK** admin_id  |
| admin_key  |

| Stt_Menu |
| ------------- | 
| **PK**    stt_id  |
| datatime |
|Breakfast |
|Lunch |
|Dinner |
| **FK** admin_id |

| Stx_Menu |
| ------------- | 
| **PK**    stx_id  |
| datatime |
|Breakfast |
|Lunch |
|Dinner |
| **FK** admin_id |

**Comment by PinkDots** [comment]
Yes, I see the issue he spoke about, lets design another table, as we might run into issues with this database

**Comment by tguadalupe** [comment]
**Cool:smiley_cat:.  Likewise. This is what I have so far:**  
![screenshot_20181123_004218](https://user-images.githubusercontent.com/44811346/48928786-a1c16780-eeb8-11e8-8dc7-6bc1204e6092.png)
**Do you understand why?**

**Comment by PinkDots** [comment]
Hm yes, this makes sense so far.  Foods with its own table and its linked to dish.  Then its saved in planned meal.  Campus table so can tell which cafeteria workers saved the menu, if its on stt or stx.   Makes good sense.
